### PR TITLE
docs: detail requirements for manual azure batch pools

### PR DESCRIPTION
**Pull request by adamrtalbot**
The workaround for Azure Batch pools did not indicate the requirements these pools had to meet. This PR references the Nextflow docs for Azure Batch named pools.

### CODE REVIEW

**Review by justinegeffen — APPROVED** [review]
Looks good from an editorial point of view. Added Ben for technical review.

**Review by bentsherman — APPROVED** [review]

File: platform_versioned_docs/version-23.3.0/faqs.mdx [hunk 1]
```diff
@@ -807,6 +807,7 @@ The default Azure Batch implementation in Seqera Platform uses a single pool for
 1. Create two Batch pools in Azure:
     - One Dedicated
     - One [Low priority](https://learn.microsoft.com/en-us/azure/batch/batch-spot-vms#differences-between-spot-and-low-priority-vms).
+    - **Note**: Both pools must meet the requirements of a pre-existing pool as detailed in the [Nextflow documentation](https://www.nextflow.io/docs/latest/azure.html#requirements-on-pre-existing-named-pools).
```
Comment (pditommaso) [inline platform_versioned_docs/version-23.3.0/faqs.mdx:810]: Relative link should be used instead

**Review by pditommaso — CHANGES_REQUESTED** [review]

File: platform_versioned_docs/version-23.3.0/faqs.mdx (hunk 1 above)
Comment (bentsherman) [reply platform_versioned_docs/version-23.3.0/faqs.mdx:810]: This is the seqera docs

Comment (pditommaso) [reply platform_versioned_docs/version-23.3.0/faqs.mdx:810]: Oops 👍

**Comment by justinegeffen** [comment]
@adamrtalbot, I'm ready to merge this. Should the 24.1.1 docs also be updated?

**Pull request merged**